src/main/java/com/adobe/cq/testing/junit/rules/ClassRuleUtils.java

```diff
@@ -41,6 +53,18 @@
         }
     }
 
+    public static class ConfigureIndexingLaneInstance extends ExistingInstance {
+        @Override
+        public <T extends SlingClient> T getClient(Class<T> clientClass, String user, String pass) {
```

Comment:
You should actually override `newClient` and `newAdminClient` which are used by `get[Admin]Client`.

Reply:
done in catholicon/aem-testing-clients@ebad5e5